storybook/src/docs/components/Typography/Typography.stories.mdx

```diff
@@ -0,0 +1,40 @@
+import { Canvas, Meta, Source, Story } from "@storybook/addon-docs";
```

Comment:
Shouldn't we document this in Docusaurus?

Reply:
I moved the docs to Docusaurus

Reply:
I don't think it's a good idea to add any component docs to Docusaurus until we find a solution for the imports, as we (@johnnyomair) discussed recently, when talking about #1869.

Otherwise we have two separate docs for our components and devs will have to guess in which (if any) a component is documented.

Reply:
@jamesricky We already have docs for FieldSet only in Docusaurus and our plan was to move all the docs. So we are in a strange place now. We should discuss this tomorrow.